src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsPrisonOffenderEventListener.kt

```diff
@@ -38,19 +39,21 @@ class IncidentsPrisonOffenderEventListener(
           val eventType = sqsMessage.MessageAttributes!!.eventType.Value
           if (eventFeatureSwitch.isEnabled(eventType)) {
             when {
-              // INCIDENT-CHANGED-CASES, INCIDENT-CHANGED-PARTIES, INCIDENT-CHANGED-RESPONSES, INCIDENT-CHANGED-REQUIREMENTS,
               eventType.startsWith("INCIDENT-CHANGED") ->
-                log.debug("Update event $eventType received {}", sqsMessage.Message.fromJson())
+                // INCIDENT-CHANGED-CASES, INCIDENT-CHANGED-PARTIES, INCIDENT-CHANGED-RESPONSES, INCIDENT-CHANGED-REQUIREMENTS,
+                incidentsSynchronisationService.synchroniseIncidentUpdate((sqsMessage.Message.fromJson()))
```

Comment:
Added call to service

Reply:
Very minor but I think `case INCIDENT-CHANGED-CASES, INCIDENT-CHANGED-PARTIES, INCIDENT-CHANGED-RESPONSES, INCIDENT-CHANGED-REQUIREMENTS ->`  would read better, `startsWith` feels unnecessarily  lazy